app/src/pages/admin/projectStatusPage/projectStatusPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { ToggleButton } from 'components/buttons/toggleButton';
+import { PERIOD_VALUES } from './constants';
+import styles from './projectStatusPage.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  sixMonths: {
+    id: 'ProjectStatusPage.sixMonths',
+    defaultMessage: '6 months',
+  },
+  threeMonths: {
+    id: 'ProjectStatusPage.threeMonths',
+    defaultMessage: '3 months',
+  },
+  oneMonth: {
+    id: 'ProjectStatusPage.oneMonth',
+    defaultMessage: '1 month',
+  },
+});
+
+@injectIntl
+export class ProjectStatusPage extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      periodValue: PERIOD_VALUES.threeMonths,
+    };
+  }
+
+  onPeriodChange = (periodValue) => {
```

Comment:
Just in single line: `onPeriodChange = (periodValue) => this.setState({ periodValue });`

Reply:
Fixed